views/ctltools/forms/delete-form.hbs

```diff
@@ -1,7 +1,34 @@
 <div class="form-group">
-    <label class="control-label" for="name">
-        Sind Sie sich sicher, dass Sie das folgende Tool löschen wollen?<br>
-        <i>Achtung: Beim Löschen des Tools wird das Tool auch in allen Kursen und Teams dauerhaft gelöscht. Bitte bestätigen Sie, dass das folgende Tool gelöscht werden soll.</i>
+    <fieldset disabled>
+        <label class="control-label" for="name">
+            Sind Sie sich sicher, dass Sie das folgende Tool löschen wollen?
+            <input type="text" style="width: 100%;" name="name" id="name">
+        </label>
+
+        <label class="control-label">
+            Zurzeit wird das Tool wie folgt genutzt:
+        </label>
+    </fieldset>
+
+    <fieldset disabled>
+        <label class="control-label" for="school">
+            <input type="text" style="max-width: 64px; text-align: end;" name="schoolExternalToolCount" id="school">
+            Schule(n)
+        </label>
+        <br>
+        <label class="control-label" for="course">
+            <input type="text" style="max-width: 64px; text-align: end;" name="contextExternalToolCountPerContext[course]" id="course">
+            Kurs(e)
+        </label>
+        <br>
+        <label class="control-label" for="board">
+            <input type="text" style="max-width: 64px; text-align: end;" name="contextExternalToolCountPerContext[boardElement]" id="board">
+            Spalten-Board(s)
+        </label>
+    </fieldset>
+
+    <label class="control-label">
+        <br>
+        <i>Achtung: Wenn das Tool entfernt wird, kann es in oben genannten Fällen nicht mehr genutzt werden. Bitte bestätigen Sie, dass das folgende Tool gelöscht werden soll.</i>
```

Comment:
"Bitte bestätigen Sie, dass das folgende Tool gelöscht werden soll." Doesnt appears in the story. Please let BA change the story if its correct otherwise change it

Reply:
That was discussed with BA. It will be added to the ticket